app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/splash_background"
+    android:theme="@style/splashScreenTheme"
+    tools:context=".SplashScreenActivity">
+
+    <ImageView
+        android:id="@+id/splash_screen_title"
+        android:layout_width="@dimen/splash_title_size"
+        android:layout_height="@dimen/splash_title_size"
+        android:layout_marginStart="@dimen/splash_title_margin"
```

Comment:
Generally you won't set big margins to certain values - depending on phone/tablet size it might look different. You can set width to 0dp, constraints start/end and use **app:layout_constraintWidth_percent** to stretch the view on a certain percetange of the entire screen (e.g when you want an image to be centered, filling 50% of the width)